Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -267,7 +267,7 @@ def audit_file
 
       problem "File should end with a newline" unless text.trailing_newline?
 
-      if formula.versioned_formula?
+      if @official_tap && formula.versioned_formula?
```

Comment:
Mind using `formula.core_formula?` instead?

Reply:
switched to `formula.core_formula?`, and there are not other instances of `@official_tap` after rebasing